src/api/authProvider.ts

```diff
@@ -0,0 +1,50 @@
+import { AuthProvider } from 'react-admin';
+import { API_URL } from '../constants';
+
+export interface LoginProps {
+  username: string;
+  password: string;
+}
+
+const authProvider: AuthProvider = {
+  login: ({ username, password }: LoginProps) => {
+    const request = new Request(`${API_URL}/auth/sign-in`, {
+      method: 'POST',
+      body: JSON.stringify({ username, password }),
```

Comment:
@munozlucas any input about this? it works fine with the strigifyed  body, so I guess it is prepared to recieve it like that

Reply:
It works fine because in this case we are using fetch API and the body data is matching to the header content-type json. I think we should use axios, I will reach you to work on this change.